### ISSUE TITLE
[Feature]: termux-file-editor

### ISSUE THREAD

**Issue by tathastu871** [termux/termux-app#2632]
### Feature description

Use termux-file-editor as termux-url-opener for note taking

### Additional information

while browsing i needed to make note of some text. when i select text and share it to termux file-editor hook is being called and every time i share new text it ask for new file name and is tedious. but if something like termux-url-opener can be used for note taking like echo "$1" >> file && exit . i can use termux for note taking

**Comment by actuariat** [comment]
I fully agree. We need an option like this in order to replicate kindle Clippings. One could make a file with all the notes he likes. Good for storing commands also. So please do something.

**Comment by tathastu871** [comment]
if termux-url-opener can run in background like .shortcuts/tasks/scripts
